.github/workflows/ci.yml

```
@@ -47,6 +47,18 @@ jobs:
     - name: Fuzz Test
       run: node Makefile fuzz
 
+  test_on_deno:
+    name: Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOs-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: denoland/setup-deno
```

```suggestion
    - uses: denoland/setup-deno@v1
```

It still doesn't work. It could be because ref is mandatory, so this might fix it.